starter-code/routes/passportRouter.js

```diff
@@ -9,6 +9,57 @@ const ensureLogin = require("connect-ensure-login");
 const passport      = require("passport");
 
 
+// SIGN UP
+router.get('/signup', (req, res) => {
+  res.render('passport/signup');
+});
+
+router.post('/signup', (req, res, next) => {
+  const username = req.body.username;
+  const password = req.body.password;
```

Comment:
Little tip: you can deconstruct `req.body` by doing `const { username, password } = req.body`.
What this neat little syntax does is that it creates two variables `username` et `password` that match the content of the `username` and `password` properties in `req.body`.